addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -594,7 +594,7 @@ const Wysiwyg = Widget.extend({
     openMediaDialog(params = {}) {
         const sel = this.odooEditor.document.getSelection();
         const fontawesomeIcon = getInSelection(this.odooEditor.document, '.fa');
-        if (fontawesomeIcon && sel.toString().trim() === "") {
+        if (fontawesomeIcon && sel.toString().trim() === "" && !$(sel.anchorNode).hasClass('o_we_user_value_widget')) {
```

Comment:
Can't be the right fix from what I understand from your commit explanation. Is this bug also there in 14.0 or is this since the new editor?

Reply:
While debugging this code, the only difference I found between document in website and iframe is that current **selection** in iframe editor starts from the option's button itself and then considered as an update of `fontawesomeIcon`

![14 3-editor-target-selection](https://user-images.githubusercontent.com/25032876/121573708-d9b43a00-ca25-11eb-91d9-c56015c62339.png)

![14 3-image-target-error](https://user-images.githubusercontent.com/25032876/121573889-05372480-ca26-11eb-98d8-aa5361128aed.png)

As a consequence, target node will be replaced in this code :

```
if (fontawesomeIcon && sel.toString().trim() === "") {
    params.node = $(fontawesomeIcon);
    ...
}
```

Which was added on 14.3 as a [fix for icon replacement](https://github.com/odoo/odoo/pull/70118/commits/756783665e07d59bef57880c0dce72ff259b6a60).

That's why I tried to exclude the 'o_we_user_value_widget' button (I reverted this commit since it should target saas-14.3 anyway).

Reply:
I opened an editor feedback for this: https://www.odoo.com/web#id=2567806&action=333&active_id=1695&model=project.task&view_type=form&cids=1&menu_id=4720 so you can ignore it